build-all.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+if [ $# -ne 2 ]
+then
+    echo "Syntax: build.sh <app_name> <tf_workspace>"
+    sleep 30
+    exit
+fi
+./build.sh $1 $2

```

Comment:
can we make this ../securityanalytics-sharedinfrastructure, I know it is the same, but it is more similar and more readily understandable to do it the same way. It names all the projects explicitly